scripts/test/test-utils.sh

```diff
@@ -116,16 +128,19 @@ run_tests() {
   echo "Converted image name: "
   echo "$converted_image_name"
 
-  python3 -m pytest --verbose \
+  eval python3 -m pytest \
```

Comment:
I added the `eval` here to be able to properly quote the filter, so that `pytest_extra_args` resolves to `-k 'not test_update'`.

However this doesn't seem to work on CI and I am running out of ideas, I need help from a shell ninja. It works locally on my machine, and CI is using bash, so I don't know where is the issue really.

Reply:
Ignore my previous comment. It actually works but I got confused with a previous run (not my last force push).